devtools/run-task.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# Common shell functions for task scripts.
+#
+# Usage:
+#   source devtools/common.sh
+#    ...  # define task functions
+#   run-task "$@"
+
+# List all functions defined in this file (and not in sourced files).
+_list-funcs() {
+    local funcs=($(compgen -A function))
+    # extdebug makes `declare -F` print the file path, but, annoyingly, only
+    # if you pass the function names as arguments.
+    shopt -s extdebug
+    declare -F "${funcs[@]}" | grep --fixed-strings " $0" | awk '{print $1}'
```

Comment:
```suggestion
    declare -F "${funcs[@]}" | grep --fixed-strings " $0" | awk '{print "  "$1}'
```
To list the tasks indented (if we don't pipe into column)